com.unity.ml-agents/Runtime/Actuators/ActuatorComponent.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+namespace Unity.MLAgents.Actuators
+{
+    /// <summary>
+    /// Editor components for creating Actuators. Generally an IActuator component should
+    /// have a corresponding ActuatorComponent.
+    /// </summary>
+    internal abstract class ActuatorComponent : MonoBehaviour
```

Comment:
I think this will need a method that returns the ActionSpec at some point (so that we can validate the size of the barrcauda model actions), but that can wait until we're ready to use it somewhere

Reply:
yes, I'll add it in a later PR.